test/integration/fixtures/blocks/core__image__deprecated-v8-deprecate-behaviors-lightbox.html

```diff
@@ -0,0 +1,3 @@
+<!-- wp:image {"lightbox":{"enabled":true},"id":8,"sizeSlug":"large","linkDestination":"none"} -->
```

Comment:
It looks like this test isn't covering the deprecated behaviors, it should rather be:

```html
<!-- wp:image {"behaviors":{"lightbox":{"enabled": true, "animation": "fade"}}},"id":8,"sizeSlug":"large","linkDestination":"none"} -->
```

The file ending with `.serialize.html` should contain the migrated version like in this file.

Reply:
Good catch, I'll update the file

Reply:
done in https://github.com/WordPress/gutenberg/pull/54570